test/QuadForm/Quad/ZLattices.jl

```diff
@@ -432,8 +432,8 @@ end
   v = [1//2]
   l = Zlattice(matrix(QQ,1,1,[1//2;]))
   @test !(x1 in L)
-  @test x2 in L
-  @test x3 in L
```

Comment:
If there are now wrong, can you remove `x2` and `x3`? Or just test `!(x2 in L)`?